agent/bench-scripts/tests/pbench-specjbb2005/test-57.opts

```diff
@@ -0,0 +1 @@
+--specjbb2005-dir=${_testtmp}/specjbb2005 --nr-jvms=2 --start-warehouses=13 --inc-warehouses=5 --stop-warehouses=23
```

Comment:
Why isn't this additional test on main?

Reply:
Seems like it is ... (see https://github.com/distributed-system-analysis/pbench/pull/2522/files#diff-5c21bbc8dae02663fbb059f64aa1f6e1f2db945ebecb1a94371a9e21dcd925d2).